programs/openbook-v2/src/state/open_orders_account.rs

```diff
@@ -331,8 +342,11 @@ pub struct Position {
     /// Cumulative taker volume in quote native units (display only)
     pub taker_volume: u128,
 
+    /// Quote lots in open bids
+    pub bids_quote_lots: i64,
```

Comment:
i wonder if this is the right way to do this. introducing this change now means that the metric will be completely unreliable for old open orders. maybe it would be more robust to introduce a version field, and only track it in >v2